cloudify_agent/tests/api/pm/test_initd.py

```diff
@@ -91,23 +91,44 @@ def test_configure_start_on_boot(self):
         daemon.configure()
 
     def test_cron_respawn(self):
-        daemon = self.create_daemon(cron_respawn=True, cron_respawn_delay=1)
+        daemon = self.create_daemon(cron_respawn=True, cron_respawn_delay=2)
         daemon.create()
         daemon.configure()
         daemon.start()
-
+        self.assertTrue(daemon.status())
+        self.logger.info('**** in test_cron_respawn daemon.status: {}'
```

Comment:
Should these perhaps be debug logs?

Reply:
I'm just playing with it and adding log messages to try and understand the problem.
The PR is not ready :)